### ISSUE TITLE
關於Win11

### ISSUE THREAD

**Issue by csgen123** [tingwei1111/maplestory-worlds-automation#6]
老師您好，想請問win11能使用嗎?
模型可以提供下載連結嗎? 非常感謝

不好意思，想順便請教老師。在訓練模型過程中，電腦效能的好壞會影響模型的準確性嗎? 還是單純的訓練時間長?

**Comment by tingwei1111** [comment]
weights 這資料夾內

**Comment by tingwei1111** [comment]
應該這說有顯卡只是能加快訓練速度我是用macbook m3 pro 所以花了點時間訓練，建議是使用N家顯卡啦 ，但我只是第一次做還有很大缺陷

**Comment by csgen123** [comment]
原來如此，非常感謝老師百忙之中抽空解答。

**Comment by tingwei1111** [comment]
不是老師啦只是在練習而已，我只在mac上跑過，如您真的要用可以直接用我勳練好檔案在自己去改一下

**Comment by csgen123** [comment]
能從您這邊學點東西，我應當得叫老師。

不好意思，想在再請教老師您一個與主題不相符的問題。

以前在網路上學習yolo v8教學，我想檢測家中長輩動向，但實際檢測結果不理想。

我製作模型使用50張照片訓練 (不理想)
又使用150張訓練 (依然不理想)

他會檢測到家中家具，或者檢測不到長輩 (我標籤，只有標記長輩)

是因為圖片還不夠嗎? 因為50與150張照片訓練出來的結果都差不多。

我的提問與主題不相符，如果老師您覺得不合適宜請將我的留言刪除或告知，我會刪除留言。

**Comment by tingwei1111** [comment]
晚點回覆您，但我覺得照片太少

	•	最基本可玩：每類200~300張
	•	較穩效果：每類500~1000張
	•	希望泛用、準確：每類2000張以上（專案等級）

當時準備訓練參考gpt

**Comment by csgen123** [comment]
我再試試看使用更多張照片訓練，非常感謝老師的回覆。

**Comment by tingwei1111** [comment]
您可以利用 Roboflow Universe 這個網站來節省大量的圖片標記時間。方法是先從平台上尋找別人已經分享、標記好的現成資料集，再根據您的專案需求，自行補充或微調一小部分的標記即可。

以我個人的**《楓之谷》**專案為例，我就是採用類似的方式，最終建立了一個包含超過 5,400 張圖片的龐大資料集。

資料集統計資訊

訓練集 (Training Set): 3,992 張圖片
驗證集 (Validation Set): 998 張圖片
測試集 (Test Set): 464 張圖片

**Comment by csgen123** [comment]
謝謝老師，我找到這個網站了。 對我學習yolo真的有很大的幫助!!
